build-system/tasks/bundle-size/README.md

```diff
@@ -0,0 +1,23 @@
+# Bundle Size checks
+
+The `APPROVERS.json` file is a listing of compiled AMP runtime and extensions
+files. Each rule has two parts:
+
+- `approvers`: list of GitHub project teams, members of which are allowed to
+  approve pull requests that fail the check
+- `threshold`: number of KBs that the file bundle size (Brotli-compressed size)
+  can increase before failing the check
+
+Approval from any single member of any of the file's approval-teams is enough to
+satisfy the check.
```

Comment:
nit: Is this sentence now implied by the reworded `approvers` text?

Reply:
Kinda, but there's nothing wrong with being extra explicit in documentation :)